FiveOhFirstDataCore/Pages/Personnel/Recruiter/AccountCreation.razor

```diff
@@ -156,16 +156,23 @@
     protected void IdChanged(ChangeEventArgs e)
     {
         string str = (string?)e.Value ?? "";
-        if(int.TryParse(str, out int val))
+        if (System.Text.RegularExpressions.Regex.IsMatch(str, @"\d\d\d\d\d"))
```

Comment:
You don't need a regex statement for checking if a string is blank. Just use `string.IsNullOrWhiteSpace(str)`.

Reply:
Corrected